deploy/Chart/templates/ucp/rbac.yaml

```diff
@@ -6,52 +6,45 @@ metadata:
     app.kubernetes.io/name: ucp
     app.kubernetes.io/part-of: radius
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - configmaps
-  - secrets
-  - services
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - apps
-  resources:
-  - deployments
-  - statefulsets
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - ucp.dev
-  resources:
-  - resources
-  - queuemessages
-  verbs:
-  - create
-  - delete
-  - get
-  - list
-  - patch
-  - update
-  - watch
-- apiGroups:
-  - api.ucp.dev
-  resources:
-  - '*'
-  verbs:
-  - '*'
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - create
+      - delete
+      - get
+      - update
+
+  - apiGroups:
+      - ucp.dev
+    resources:
+      - resources
+    verbs:
+      - create
+      - delete
+      - get
+      - list
+      - update
+
+  - apiGroups:
+      - ucp.dev
+    resources:
+      - queuemessages
+      - resources
```

Comment:
Resources is duplicated here. 

- https://github.com/radius-project/radius/pull/8080/files#diff-24ef6f7a71ac496e97b570c9d42f82358cbd496e660eb058079543cc05ff1805R34
- https://github.com/radius-project/radius/pull/8080/files#diff-24ef6f7a71ac496e97b570c9d42f82358cbd496e660eb058079543cc05ff1805R22